server/app.py

```diff
@@ -17,12 +17,17 @@
 db = SQLAlchemy(app) 
 jwt = JWTManager(app)
 
-from resources import UserResources, GmailResources
+from models import UserModel, ProspectModel, TagModel, CampaignModel
+from resources import UserResources, GmailResources, CampaignResources
 
 api.add_resource(UserResources.UserRegister, '/register')
 api.add_resource(UserResources.UserLogin, '/login')
+api.add_resource(UserResources.UserCampaigns, '/campaigns')
 api.add_resource(GmailResources.GetAuthURL, '/gmail/get_auth_url')
 api.add_resource(GmailResources.Authorize, '/gmail/authorize')
 api.add_resource(GmailResources.GetGmailAddress, '/gmail/get_address')
+api.add_resource(CampaignResources.NewCampaign, '/campaign/new')
+api.add_resource(CampaignResources.CampaignProspects, '/campaign/<int:id>/prospects')
+api.add_resource(CampaignResources.AddProspectsToCampaign, '/prospects/campaign')
```

Comment:
same as above - adding prospects can be a post in /campaign/<int:id>/prospects resource

Reply:
Passing the id parameter in both the url and inside the request body? Wouldn't that be redundant? Shouldn't a POST request not have parameters being sent in the url?

Reply:
Or can i just do /campaign/prospects in a post request?

Reply:
Yeah you can do something like /campaign/prospects - yes if you pass it in the url, do you don't need to pass it into the request body - the int:id would be of the campaign id